packages/rocketchat-grant/server/index.js

```diff
@@ -12,9 +13,30 @@ import Providers, { middleware as providers } from './providers';
 import Settings from './settings';
 
 let grant;
+let storeAddress = process.env.SESSIONS_URL || 'mongodb://localhost:27017/rocketsession';
```

Comment:
any reason to use `SESSIONS_URL` instead of default Rocket.Chat's Mongo connection? In other words, any reason to not store this data along with regular Rocket.Chat data?

Reply:
First reason I would see: I have one cluster in DC1, another in DC2, maybe others, using geo DNS/betting clients won't fail over from one to another unless DC failure, then I don't need to distribute sessions over my two locations, only RocketChat data.
Then again, I could be over-thinking it.